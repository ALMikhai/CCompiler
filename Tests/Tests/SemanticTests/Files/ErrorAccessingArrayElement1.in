int f() {
    int a[1];
    int b = a[1.0];
}