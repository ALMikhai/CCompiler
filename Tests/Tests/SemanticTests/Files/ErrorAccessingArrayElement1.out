(3:15): semantic error: array subscript is not an integer
